src/Evaluator.ml

```diff
@@ -29,63 +30,77 @@ let refresh_g idg fvs =
 (* modal (modal) substitution *)
 let rec subst_m term idg body =
```

Comment:
```suggestion
let rec subst_m term idg Location.{data = body; loc} =
```

Reply:
Hey, about this method, if I understand correctly, Ocaml doesn't have a way to make the Argument Capture, i mean: `var@Location.{data = body; look}`, so I did it other, please check